release/authorization/README.md

```diff
@@ -0,0 +1,420 @@
+  }
+  action: PERMIT
+  level: READ
+  user { name: "stevie" }
+}
+# Action members of family-group to access /this/is/a/different/message_path in
+# both READ and WRITE methods.
+policy {
+  id: "two"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem { name: "different" }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  level: WRITE
+  group { name: "family-group" }
+}
+# Demonstrate READ access to a key with an attribute defined.
+policy {
+  id: "key"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem {
+      name: "keyed"
+      key{
+        key: "Ethernet"
+        value: "1/2/3"
+      }
+    }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  group { name: "test-group" }
+}
+# Demonstrate READ access to a key with a wildcard attribute.
+policy {
+  id: "wyld"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem {
+      name: "keyed"
+      key{
+        key: "Ethernet"
+        value: "*"
+      }
+    }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  group { name: "family-group" }
+}
+# Demonstrate a key with a wildcard attribute and a user specific match.
+# The previous policy matches all family-group users and permits a command
+# path, the policy rule below specifically denies brian access to these paths.
+policy {
+  id: "wyld-stallions"
+  log_level: LOG_BRIEF
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem {
+      name: "keyed"
+      key{
+        key: "Ethernet"
+        value: "*"
+      }
+    }
+    elem { name: "message_path" }
+  }
+  action: DENY
+  level: READ
+  group { name: "brian" }
+}
+# Add a final rule which is an explicit deny rule.
+policy {
+  id: "explicit-deny"
+  log_level: LOG_FULL
+  level: READ
+  level: WRITE
+  action: DENY
+}
+```
+
+The example first policy rule:
+
+```proto
+# Action stevie to access /this/is/a/message_path for READ.
+policy {
+  id: "one"
+  log_level: LOG_NONE
+  path {
+    origin: "foo"
+    elem { name: "this" }
+    elem { name: "is" }
+    elem { name: "a" }
+    elem { name: "message_path" }
+  }
+  action: PERMIT
+  level: READ
+  user { name: "stevie" }
+}
+```
+
+permits the singular user "stevie" to access the path:
+
+```shell
+    /this/is/a/message_path
+```
+
+Additionally, "stevie" is permitted access to all paths below the defined path,
+in a READ only mode, such as:
+
+```shell
+    /this/is/a/message_path/the
+    /this/is/a/message_path/the/one
+    /this/is/a/message_path/the/one/that
+    /this/is/a/message_path/the/one/that/knocks
+```
+
+The second policy rule:
+
+```proto
+# Action memberss of family-group to run /this/is/a/different/message_path
```

Comment:
s/memberss/members

Reply:
done.

(why can't vim do this for me.. it probably can! I probably disabled it because it's annoying! :) )